Sources/Plasma/Apps/plClient/main.cpp

```diff
@@ -59,5 +64,9 @@ static plClientLoader gClient;
 // Stub main function so it compiles on non-Windows
 int main(int argc, const char** argv)
 {
+#ifdef PLASMA_PIPELINE_GL
+    hsG3DDeviceSelector::AddDeviceEnumerator(&plGLEnumerate);
```

Comment:
I think this would be better to do in some static object defined in `pfGLPipeline`, preventing this bit of boilerplate from needing to be duplicated into all client main implementations.

Reply:
yeah, I definitely don't like the duplication.

I wish C++ would do something like Java's `static { /* static init stuff here */ }` blocks rather than having to fake it with constructors of static objects 😞